requirements.txt

```diff
@@ -4,6 +4,8 @@ django-filter==2.0.0
 djangorestframework==3.9.0
 docx-mailmerge==0.4.0
 docxtpl==0.5.14
+mysqlclient==1.3.13
+psycopg2-binary==2.7.6.1
```

Comment:
Duplicate requirement

Reply:
ouch. That was still there from the cookiecutter. Removed.